api/layer/container.go

```diff
@@ -130,19 +135,15 @@ func (n *layer) createContainer(ctx context.Context, p *CreateBucketParams) (*ci
 		return nil, err
 	}
 
-	if err := n.setContainerEACL(ctx, cid, p.BoxData.Gate.GateKey); err != nil {
+	if err := n.setContainerEACLTable(ctx, cid, p.EACL); err != nil {
```

Comment:
Did I understand correctly, that default eACL will be set up on higher lever (with the help of `addPredefinedACP`) and then passed as `p.EACL`?

Reply:
Yes. We set up default eACL (initialy as `AccessControlPolicy`) in `parseACLHeaders` function. And if `X-Amz-Acl` is provided we complement eACL in `addPredefinedACP` function.